docs/resources/Emoji.md

```diff
@@ -16,7 +16,7 @@
 | require_colons? | boolean                                                          | whether this emoji must be wrapped in colons                              |
 | managed?        | boolean                                                          | whether this emoji is managed                                             |
 | animated?       | boolean                                                          | whether this emoji is animated                                            |
-| available?      | boolean                                                          | whether this emoji can be used, may be false due to loss of Server Boosts |
+| available?      | boolean                                                          | whether this emoji can be used may be false due to loss of Server Boosts |
```

Comment:
this is not one sentence and needs a separator. might make more sense to you if you put a `;` instead of the comma, which probably looks better anyway